charts/drupal/values.yaml

```diff
@@ -470,6 +470,15 @@ mariadb:
         memory: 384M
       limits:
         memory: 512M
+    extraInitContainers: |
+      - name: dbinfo
+        image: busybox:stable
+        command: ["/bin/sh", "-c"]
+        args:
+          - df -h /bitnami/mariadb;
```

Comment:
Is the semicolon at the end of the line needed?

Reply:
can remove it as well